src/promptflow/promptflow/_sdk/entities/_flow.py

```diff
@@ -266,10 +302,10 @@ def __call__(self, *args, **kwargs):
 
         if args:
             raise UserErrorException("Flow can only be called with keyword arguments.")
-        with TestSubmitter(flow=self, flow_context=self.context).init() as submitter:
-            # validate inputs
-            flow_inputs, _ = submitter.resolve_data(inputs=kwargs)
-            result = submitter.exec_with_inputs(
-                inputs=flow_inputs,
-            )
+
+        submitter = TestSubmitter(flow=self, flow_context=self.context)
```

Comment:
Do we need to create this everytime? I guess maybe we can hold one executor in the `_flow` object instead of doing the cache?

Reply:
The executor is more related to operations(it needs connections to initialize), not sure if we want to couple entity with operation.
Also have a little concern if user changed context of a flow's context, should we update the executor in flow?

Reply:
Will keep this implementation for now since we'll have same flow implementation for different executors(Python/C#).